.gitignore

```diff
@@ -1,3 +1,4 @@
-/target
-Cargo.lock
+target
+/Cargo.lock
```

Comment:
Does it really have to add other lock files?

Reply:
its used by the cache it would work without tho

Reply:
fixed